### ISSUE TITLE
auto upgrade extensions installed via an npm package

### ISSUE THREAD

**Issue by jstrachan** [lensapp/lens#3579]
if you install an extension via the npm package format such as using `@jenkins-x-plugins/jx-lens` instead of the URL to a specific version it would be awesome if Lens would periodically check for upgrades (e.g. on startup? or when upgrading Lens?)

**Comment by Nokel81** [comment]
You can already do this, just put in `@jenkins-x-plugins/jx-lens` and the latest version from npm will install.

**Comment by jstrachan** [comment]
@Nokel81 ah thats awesome! we should upgrade the docs to let folks know!

When does Lens decide to upgrade; just when you do the first install?

**Comment by Nokel81** [comment]
Do you mean does lens ever decide to upgrade an extension? The answer to that is never, we have not implemented an update checker for extensions.

**Comment by jstrachan** [comment]
@Nokel81 yeah, I was interested in the auto-upgrade capability. Thanks for the feedback!
